app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= link_to 'New Status Update', [:new, :students, :status_update], class: 'btn btn-default'
+
+p
```

Comment:
If you need some extra spacing I'd add a class to it and style that using CSS. Alternatively use a `br` tag, using an empty paragraph element is a hack.